handler/core/src/test/java/io/knotx/fragments/task/factory/GraphNodeOptionsTest.java

```diff
@@ -86,6 +87,16 @@ void expectTransitionSuccessWithNodeBThenNodeC(Vertx vertx) throws Throwable {
     }, vertx);
   }
 
+  @Test
+  @DisplayName("Expect 'on' to alias 'onTransitions'")
```

Comment:
Please add the case when we mix `onTransitions` and `on`.

Additionally please update all functional tests to use `on` instead od `onTransitions`.